### DISCUSSION TITLE
创建 StatefulSet - DaoCloud Enterprise

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#848] [Announcements]
# 创建 StatefulSet - DaoCloud Enterprise

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/kpanda/07UserGuide/Workloads/CreateStatefulSetByImage/

**Comment by liyuerich** [comment]
网络配置中能否增加 容器网卡 的配置？

**Reply by qiupingdai** [reply]
https://docs.daocloud.io/network/modules/spiderpool/usage/#_2  可以参考这篇文档，上述文档待更新

**Reply by yanrongshi** [reply]
需要符合前提条件：
![image](https://github.com/DaoCloud/DaoCloud-docs/assets/29862632/a61cd1ce-25c7-4825-843d-780bf22e64a2)

**Reply by Michelle951** [reply]
满足上图中的前提条件后，网络配置页签下会出现配置容器网卡的选项。具体参数配置说明可参考：https://docs.daocloud.io/network/modules/spiderpool/usage/#_2 

本文档也已经过优化，解决其他读者可能产生的类似疑问，见 pull request：https://github.com/DaoCloud/DaoCloud-docs/pull/1885
